src/openvic-simulation/politics/Issue.cpp

```diff
@@ -157,22 +190,22 @@ bool IssueManager::load_issues_file(ast::NodeCPtr root) {
 	issues.reserve(issues.size() + expected_issues);
 	reforms.reserve(reforms.size() + expected_reforms);
 
-	ret &= expect_dictionary([this](std::string_view type_key, ast::NodeCPtr type_value) -> bool {
-		return expect_dictionary([this, type_key](std::string_view group_key, ast::NodeCPtr group_value) -> bool {
+	ret &= expect_dictionary([this, &modifier_manager, &rule_manager](std::string_view type_key, ast::NodeCPtr type_value) -> bool {
+		return expect_dictionary([this, &modifier_manager, &rule_manager, type_key](std::string_view group_key, ast::NodeCPtr group_value) -> bool {
 			if (type_key == "party_issues") {
 				IssueGroup const* issue_group = get_issue_group_by_identifier(group_key);
-				return expect_dictionary([this, issue_group](std::string_view key, ast::NodeCPtr value) -> bool {
-					bool ret = _load_issue(key, issue_group, value);
+				return expect_dictionary([this, &modifier_manager, &rule_manager, issue_group](std::string_view key, ast::NodeCPtr value) -> bool {
+					bool ret = _load_issue(modifier_manager, rule_manager, key, issue_group, value);
 					return ret;
 				})(group_value);
 			} else {
 				ReformGroup const* reform_group = get_reform_group_by_identifier(group_key);
 				size_t ordinal = 0;
-				return expect_dictionary([this, reform_group, &ordinal](std::string_view key, ast::NodeCPtr value) -> bool {
+				return expect_dictionary([this, &modifier_manager, &rule_manager, reform_group, &ordinal](std::string_view key, ast::NodeCPtr value) -> bool {
 					if (key == "next_step_only" || key == "administrative") {
 						return true;
 					}
-					bool ret = _load_reform(ordinal, key, reform_group, value);
+					bool ret = _load_reform(modifier_manager, rule_manager, ordinal, key, reform_group, value);
 					ordinal++;
 					return ret;
```

Comment:
```suggestion
					return _load_reform(modifier_manager, rule_manager, ordinal++, key, reform_group, value);
```
This will increase `ordinal` by 1, but use the value before incrememting as the function argument (as opposed to `++ordinal`, which would use the incremented value as the argument).

Reply:
Will do, I knew about this but I tend to avoid these things for the sake of clarity.